dso-l2/src/main/java/com/tc/objectserver/handler/RetirementManager.java

```diff
@@ -190,16 +185,16 @@ public synchronized void entityWasDestroyed() {
     Assert.assertTrue(this.waitingForDeferredRegistration.isEmpty());
   }
   
-  public Map<String, Object> getState() {
+  public synchronized Map<String, Object> getState() {
```

Comment:
doesn't this block request processor threads?

Reply:
yes but the alternative is a random CME.  We should look into refactoring this class to be more concurrent at some point.